### PR TITLE
docs: Update Japanese translation 2019009

### PR DESCRIPTION
**Pull request by yoshwata**
## Context

Update Japanese translation about [this comparison](https://github.com/screwdriver-cd/guide/compare/705f11a...master)

## Objective

Update Japanese translation.

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: docs/ja/user-guide/configuration/annotations.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ jobs:
 
 ## パイプラインレベルのアノテーション
 
-パイプラインレベルのアノテーションはパイプライン全体の設定を変更するために利用します。パイプラインレベルのアノテーションの指定箇所はジョブや`shared`と同じです。
+パイプラインレベルのアノテーションはパイプライン全体の設定を変更するために利用します。パイプラインレベルのアノテーションの指定箇所はジョブや`shared`と同じです。プルリクエストは、パイプラインのSCMブランチではないため、これらのアノテーションの変更はできません。
```
Comment (kumada626) [inline docs/ja/user-guide/configuration/annotations.md]: 以下の感じの訳ではどうでしょうか。
```suggestion
パイプラインレベルのアノテーションはパイプライン全体の設定を変更するために利用します。パイプラインレベルのアノテーションの指定箇所はジョブや`shared`と同じです。これらのアノテーションはパイプラインのSCMブランチで設定される必要があるため、プルリクエストでは変更できません。
```

Comment (yoshwata) [reply docs/ja/user-guide/configuration/annotations.md]: いただきました！

**Review by kumada626 — APPROVED** [review]
LGTM

**Review by wahapo — APPROVED** [review]
👍

**Review by tkyi — APPROVED** [review]

**Pull request merged**